hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/RawLocalFileSystem.java

```diff
@@ -518,7 +520,12 @@ public boolean delete(Path p, boolean recursive) throws IOException {
     }
     return new FileStatus[] {
         new DeprecatedRawLocalFileStatus(localf,
-        getDefaultBlockSize(f), this) };
+        defaultBlockSize, this) };
+  }
+
+  @Override
+  public boolean exists(Path f) throws IOException {
+    return pathToFile(f).exists();
```

Comment:
is this a new method, or just an accidental patch quirk

Reply:
This is a new method(Overriden) for this patch.